client/blocks/reader-subscription-list-item/email-settings.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import Popover from 'components/popover/index';
+import SegmentedControl from 'components/segmented-control';
+import ControlItem from 'components/segmented-control/item';
+import FormToggle from 'components/forms/form-toggle/compact';
+
+class ReaderEmailSubscriptionSettingsPopout extends Component {
+	static displayName = 'ReaderEmailSubscriptionSettingsPopout';
+	static propTypes = {
+		feedId: PropTypes.number,
+		siteId: PropTypes.number,
+	};
+	state = {
+		showPopover: false,
+	};
+
+	togglePopoverVisibility = () => {
+		this.setState( { showPopover: ! this.state.showPopover } );
+	}
+
+	closePopover = () => {
+		this.setState( { showPopover: false } );
+	}
+
+	savePopoutSpanRef = spanRef => {
+		this.spanRef = spanRef;
+	}
+
+	setSelected = text => () => {
+		this.setState( { selected: text } );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<span
+					className="reader-subscription-list-item__settings-menu"
+					onClick={ this.togglePopoverVisibility }
+					ref={ this.savePopoutSpanRef }
+				>
+					<Gridicon icon="cog" size={ 20 } />
+					Settings
+				</span>
+
+				<Popover
+					onClose={ this.closePopover }
+					isVisible={ this.state.showPopover }
+					context={ this.spanRef }
+					position={ 'bottom left' }
+					className="reader-subscription-list-item__settings-menu-popout"
+				>
+					<div className="reader-subscription-list-item__email-popout-wrapper">
+						<h3 className="reader-subscription-list-item__email-popout-header">
+							{ translate( 'Email me' ) }
+						</h3>
+						<hr className="reader-subscription-list-item__header-separator" />
+						<div className="reader-subscription-list-item__email-popout-toggle">
+							{ translate( 'New posts' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'new posts', { context: 'text for multiple new posts notification (2 new posts)'} )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/settings-popout)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).